<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdfManager</class>
 <widget class="QDialog" name="PdfManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export PDFÂ background Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset theme="pdf">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pdf path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_pathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_browseFile">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="m_resoGroupBox">
         <property name="title">
          <string>Size:</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_widthBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Height: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_heightBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_grayScale">
         <property name="text">
          <string>Grayscale</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Pages</string>
       </property>
       <widget class="QCheckBox" name="m_pagesCb">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>100</y>
          <width>66</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Pages</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="m_allPagesCb">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>42</y>
          <width>87</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>All pages</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="m_currentPageCb">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>71</y>
          <width>115</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Current page</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_pageLine">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>81</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPdfView" name="m_pdfView" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPdfView</class>
   <extends>QWidget</extends>
   <header location="global">QPdfView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PdfManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>826</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PdfManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>826</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pagesCb</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_pageLine</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
